x/crosschain/keeper/cctx_orchestrator_validate_inbound.go

```diff
@@ -48,3 +56,44 @@ func (k Keeper) ValidateInbound(
 
 	return &cctx, nil
 }
+
+// CheckIfMigrationDeposit checks if the sender is a TSS address and returns an error if it is.
+// If the sender is an older TSS address, this means that it is a migration transfer, and we do not need to treat this as a deposit and process the CCTX
+func (k Keeper) CheckIfMigrationDeposit(ctx sdk.Context, msg *types.MsgVoteInbound) error {
```

Comment:
If not yet, we should have the ZetaClient directly checking for this and skipping voting inbounds. We should create an issue for it.

Reply:
https://github.com/zeta-chain/node/issues/2486